theme_anelusia/views/snippets/s_company_team_basic.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_basic" inherit_id="website.s_company_team_basic">
```

Comment:
Consistency. Some inherited (see Be wise theme) templates have a name attribute and some not.